scripts/microbe_stage/organelle_table.as

```diff
@@ -104,6 +104,9 @@ class OrganelleParameters{
 
     //! Cost in mutation points
     int mpCost = 0;
+
+    //!  boolean used in editor to know if organelle that player is try to putneed presence or not of nucleus.
```

Comment:
There are extra spaces here. Also I'd like comments to be started with a capital letter.